### PR TITLE
Fix async mode for select component

### PR DESCRIPTION
**Pull request by x8lucas8x**

### CODE REVIEW

File: src/forms/SelectField.js [hunk 1]
```diff
@@ -6,36 +6,31 @@ import Select from '../ui/Select';
...
-  creatable,
-  async,
   onChangeValue,
   innerRef,
   onChangeWithValue = true,
   denormalize,
   ...props
 }) => {
-  const denormalized = (denormalize ? denormalize(input.value) : input.value) || defaultValue;
+  const denormalized = (!_.isNil(denormalize) ? denormalize(input.value) : input.value) || defaultValue;
```
Comment (twheys) [inline src/forms/SelectField.js]: could you put a comment here to explain what this code is doing?

File: src/ui/Abbreviation.js [hunk 2]
```diff
@@ -2,18 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 /**
- * Used to wrap a label in abbreviation tags if a valid tooltip is provided. We differentiate the abbr style
- * depending on the type passed in so that the dotted line is shown properly on the metric labels.
+ * Used to wrap a label in abbreviation tags if a valid tooltip is provided.
  */
 const Abbreviation = ({
-  label, type, tooltip, ...props
+  label, bottomBorderColor, tooltip, ...props
```
Comment (twheys) [inline src/ui/Abbreviation.js]: would probably make more sense to take a style prop.

File: src/ui/Abbreviation.js [hunk 3]
```diff
@@ -2,18 +2,17 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 /**
- * Used to wrap a label in abbreviation tags if a valid tooltip is provided. We differentiate the abbr style
- * depending on the type passed in so that the dotted line is shown properly on the metric labels.
+ * Used to wrap a label in abbreviation tags if a valid tooltip is provided.
  */
 const Abbreviation = ({
```
Comment (twheys) [inline src/ui/Abbreviation.js]: Is there much value in offering this as a component in bandit-pouch? Seems to just wrap an html element and not add much more functionality. <abbr> is styled automatically by bootstrap.

File: src/forms/SelectField.js (hunk 1 above)
Comment (x8lucas8x) [reply src/forms/SelectField.js]: @twheys sure. That came from dashmore's select field btw. That said there is no usage of it on Bandit. So, if you're not using it in Dashmore I'd rather remove the prop.

File: src/ui/Abbreviation.js (hunk 3 above)
Comment (x8lucas8x) [reply src/ui/Abbreviation.js]: @twheys I don't have a strong case for that one. It's used in a couple of places in Bandit, but I guess we can leave that out. I'll remove it.

**Pull request merged**